Dockerfile

```diff
@@ -1,50 +1,50 @@
-FROM ubuntu:trusty
+FROM isuper/java-oracle:jdk_8
```

Comment:
I did that code something like 2 years ago, so I guess OpenJDK wasn't available. From a trademark and license perspective, it's probably better to use OpenJDK, but I don't know if it works well enough, or if it will cause bugs.